Documentation/Floor.md

```diff
@@ -0,0 +1,28 @@
+# Why do we use Floor.h?
```

Comment:
Why ```Floor.h```? Can I use this library with C or it's only C++?
We need a pattern. I recommend hpp, because we are using C++. :warning: